### ISSUE TITLE
Parsing error when using `render locals` with Ruby 3.1 shorthand hash syntax

### ISSUE THREAD

**Issue by Timmitry** [presidentbeef/brakeman#1709]
### Background

Brakeman version: 5.2.2
Rails version: 6.1
Ruby version: 3.1

Link to Rails application code: -

### Introduction

I'm getting many errors of the form

```
Error: undefined method `original_line' for nil:NilClass          unless value.original_line                     ^^^^^^^^^^^^^^
Location: /Users/tim.landwerth/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/brakeman-5.2.2/lib/brakeman/processors/lib/render_helper.rb:129:in `block in process_template'
```

when running brakeman on my rails project. The same error was mentioned in #1700, however, that was already fixed in Brakeman 5.2.2 and had to with Brakeman not being able to parse ruby 3.1 hash shorthand syntax at all.

The stacktrace does not yield any helpful information, so I dug into the code a bit. The line the error comes from is

https://github.com/presidentbeef/brakeman/blob/0be81f6056b8e24f05da29546379fb079bdca573/lib/brakeman/processors/lib/render_helper.rb#L125-L133

I put a `puts "VARIABLE: #{_var}, VALUE: #{value}"` before line 128. This then outputs something like

```
VARIABLE: s(:call, nil, :filters), VALUE: s(:call, nil, :filter_params)
VARIABLE: s(:call, nil, :filter_options), VALUE: 
undefined method `original_line' for nil:NilClass
```

This i could then backtrace to my application code, where I do the following:

```
render locals: {
  filters: filter_params,
  filter_options:,
}
```

Note the hash shorthand syntax for `filter_options`, which raises the issue, while `filter: filter_params` is working fine (and corresponds to the first line of log output above).

If I change line 102 here:

https://github.com/presidentbeef/brakeman/blob/0be81f6056b8e24f05da29546379fb079bdca573/lib/brakeman/processors/lib/render_helper.rb#L99-L105

to use `= value || key`, this fixes the issue (but I'm not sure if this is a real fix, as the value could be `nil`, `false`, ...).

I hope this is enough information for a fix, but let me know if you have any questions!

**Comment by jcowhigjr** [comment]
@Timmitry I'm curious if this is fixed for you when you pull the gem from github directly?  

https://github.com/presidentbeef/brakeman/compare/v5.2.2...main

I get a bundle audit issue when doing this so for now I'm just not going to use the shorthand nested local syntax like you mention and wait for a release.

**Comment by presidentbeef** [comment]
This was fixed in #1700
